routes/eli.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response } from 'express';
+import { getEliData } from '../lib/utils';
+import { OPARL_ENDPOINT } from '../config';
+
+const router = express.Router();
+
+router.get('/*', async (req: Request, res: Response) => {
```

Comment:
maybe add a comment here saying that this is meant as a proxy in front of an opral endpoint that translates its result to eli